databricks/scripts/driver_only.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+cat <<EOF > /tmp/start_datadog.sh
+#!/bin/bash
+
+date -u +"%Y-%m-%d %H:%M:%S UTC"
+echo "Running on the driver? \$DB_IS_DRIVER"
+echo "Driver ip: \$DB_DRIVER_IP"
+
+DB_CLUSTER_NAME=$(echo "$DB_CLUSTER_NAME" | sed -e 's/ /_/g' -e "s/'/_/g")
+
+if [[ \${DB_IS_DRIVER} = "TRUE" ]]; then
+  echo "Installing Datadog Agent on the driver..."
+
+  # CONFIGURE HOST TAGS FOR DRIVER
+  DD_TAGS="environment:\${DD_ENV}","databricks_cluster_id:\${DB_CLUSTER_ID}","databricks_cluster_name:\${DB_CLUSTER_NAME}","spark_host_ip:\${DB_DRIVER_IP}","spark_node:driver"
```

Comment:
Could we add other databricks env variables as tags, like `DB_INSTANCE_TYPE` or `DB_IS_JOB_CLUSTER` from https://docs.databricks.com/clusters/init-scripts.html#environment-variables

Reply:
added!